Dockerfile

```diff
@@ -4,7 +4,7 @@ LABEL application="artbackend"
 
 
 ARG SECRET_KEY='secret key'
-ARG DJANGO_SETTINGS_MODULE='settings.prod'
+ARG DJANGO_SETTINGS_MODULE='settings.dev'
```

Comment:
The change to `settings.dev` may make docker containers built for production use the dev settings.